src/service/PermissionManager.cpp

```diff
@@ -10,17 +10,16 @@ namespace nebula {
 namespace graph {
 
 // static
-bool PermissionManager::canReadSpace(Session *session, GraphSpaceID spaceId) {
+Status PermissionManager::canReadSpace(Session *session, GraphSpaceID spaceId) {
     if (!FLAGS_enable_authorize) {
-        return true;
+        return Status::OK();
     }
     if (session->isGod()) {
-        return true;
+        return Status::OK();
     }
-    bool havePermission = false;
     auto roleResult = session->roleWithSpace(spaceId);
     if (!roleResult.ok()) {
-        return havePermission;
+        return Status::PermissionError("No permission to read space %d.", spaceId);
```

Comment:
return the space name, the error msg return to user.

Reply:
Fixed.

Reply:
It doesn't solve